webapp-django/crashstats/crashstats/utils.py

```diff
@@ -550,7 +550,7 @@ def build_default_context(product_name=None, versions=None):
 
     # Build product information
     all_products = productlib.get_products()
-    context["products"] = [product.name for product in all_products]
+    context["products"] = all_products
```

Comment:
Now "products" is a list of Product instances. That simplifies some templates and other things.